src/main/scala/com/reagroup/appliedscala/urls/fetchenrichedmovie/EnrichedMovie.scala

```diff
@@ -36,4 +36,13 @@ object EnrichedMovie {
     * Hint: You will need to create a custom encoder (you can use .forProduct).
     */
 
+  implicit val encoder: Encoder[EnrichedMovie] = (enrichedMovie: EnrichedMovie) => {
+    Json.obj(
+      ("name", Json.fromString(enrichedMovie.movie.name)),
```

Comment:
Answer key did this:
```scala
    Json.obj(
      "name" := enrichedMovie.movie.name,
      "synopsis" := enrichedMovie.movie.synopsis,
      "reviews" := enrichedMovie.movie.reviews,
      "metascore" := enrichedMovie.metascore
    )
```

I did not end up using the hint `forProduct`. Neither did the answer key. 🤷‍♀️